openstudiocore/resources/model/OpenStudio.idd

```diff
@@ -5992,14 +5992,31 @@ OS:OtherEquipment,
        \reference SpaceItemNames
        \reference SpaceLoadNames
        \reference SpaceComponentInstanceNames
-  A3, \field Other Equipment Definition Name
+  A3,  \field Other Equipment Definition Name
        \type object-list
        \required-field
        \object-list OtherEquipmentDefinitionNames
-  A4, \field Space or SpaceType Name
+  A4,  \field Fuel Type
```

Comment:
I say move it to the end. This already is divergent from the EnergyPlus object (object name). Maybe @kbenne has different feelings about it.